lib/referencer.js

```diff
@@ -453,6 +459,19 @@ class Referencer extends esrecurse.Visitor {
         this.visitProperty(node);
     }
 
+    PropertyDefinition(node) {
+        const { computed, key, value } = node;
+
+        if (computed) {
+            this.visit(key);
+        }
+        if (value) {
+            this.scopeManager.__nestClassFieldInitializerScope(value);
+            this.visit(value);
+            this.close(value);
```

Comment:
Just a non-blocking observation: the class field initializer scope might have been already closed at this point, triggered by closing a child function/class scope on the same node, but then this will simply do nothing as the current scope is already set to the upper class scope.